package.json

```diff
@@ -13,6 +15,11 @@
   "categories": [
     "Programming Languages"
   ],
+  "keywords": [
+    "swift",
+    "sswg"
```

Comment:
Just wondering if this does anything? I don't assume anyone will ever explicitly search for `sswg`, and if they do, most likely it's unrelated to Swift?

Reply:
SSWG at the moment is the publisher name, we can remove at a later date if deemed unnecessary